### PR TITLE
change docker-compose to api-only

### PR DESCRIPTION
**Pull request by greentornado**
Modify docker-compose file to start with indivious api only

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,21 +1,12 @@
...
       context: .
       dockerfile: docker/Dockerfile
     restart: unless-stopped
     ports:
       - "127.0.0.1:3000:3000"
-    depends_on:
-      - postgres
 
 volumes:
   postgresdata:
```
Comment (omarroth) [inline docker-compose.yml]: This should also be removed, otherwise it will create a default network, etc. for unneeded `postgresdata`.

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,5 @@
 /invidious
 /sentry
 shard.lock
+invidious.o
+invidious.dwarf
```
Comment (omarroth) [inline .gitignore]: This seems like it should go in it's own PR.

**Comment by omarroth** [comment]
Thanks! Left a couple comments, otherwise LGTM :+1:.

**Comment by greentornado** [comment]
Yeah I made changes followed to your comments @omarroth

**Comment by omarroth** [comment]
Thanks!
